classifiers/grid_search.py

```diff
@@ -19,22 +19,23 @@
 import engineer
 import lib
 
+from pprint import pprint
```

Comment:
Used pprint to print the complete list of parameters for all classifiers at the end